PWGHF/Core/HfMlResponseXicToPKPi.h

```diff
@@ -129,9 +142,8 @@ class HfMlResponseXicToPKPi : public HfMlResponse<TypeOutputScore>
   /// \param prong1 is the candidate's prong1
   /// \param prong2 is the candidate's prong2
   /// \return inputFeatures vector
-  template <typename T1, typename T2>
-  std::vector<float> getInputFeatures(T1 const& candidate,
-                                      T2 const& prong0, T2 const& prong1, T2 const& prong2, bool const& caseXicToPKPi)
+  template <typename T1>
+  std::vector<float> getInputFeatures(T1 const& candidate, bool const& caseXicToPKPi)
```

Comment:
I don't see `caseXicToPKPi` changed anywhere.
```suggestion
  std::vector<float> getInputFeatures(T1 const& candidate, bool const caseXicToPKPi)
```

Reply:
changing now

Reply:
fixed for all the 3 cases